pyspectral/atm_correction_ir.py

```diff
@@ -128,12 +129,29 @@ def delta(z):
         DELTA_REF = 6.2
         return (1 + DELTA_REF)**ratio(z, Z_0, Z_REF) - 1
 
-    y0, x0 = np.ma.where(view_zen == 0)
-    data[y0, x0] += tau0(data[y0, x0])
+    if hasattr(data, 'compute') or hasattr(view_zen, 'compute'):
+        is_dask_data = True
+    else:
+        is_dask_data = False
 
-    y, x = np.ma.where((view_zen > 0) & (view_zen < 90) & (~data.mask))
-    data[y, x] += tau(data[y, x]) * delta(view_zen[y, x])
-    return data
+    if is_dask_data:
```

Comment:
Code factor (the bot) makes some good points. You could replace the above `is_dask_data` with a single line (ex. `is_data_data = hasattr(data, 'compute') ...`.

Reply:
No problem, I will change it.